vivisect/tests/testvivisect.py

```diff
@@ -450,14 +450,23 @@ def test_vaset_populate(self):
                                 (134563178, 'jmp dword [0x0805e250 + edi * 4]', 65545),
                                 (134569031, 'jmp dword [0x0805e4ac + edi * 4]', 65545)],
             'NoReturnCalls': [(134573120,), (134524676,), (134577802,), (134582295,), (134551055,), (134525456,),
-                              (134568495,), (134534039,), (134524568,), (134528303,), (134578863,), (134577311,),
-                              (134549928,), (134544175,), (134553138,), (134572852,), (134580661,),
+                              (134568495,), (134534039,), (134528303,), (134578863,), (134577311,),
+                              (134549928,), (134544175,), (134553138,), (134572852,),
                               (134550582,), (134579529,), (134572857,), (134534204,), (134577471,), (134580544,),
                               (134568769,), (134546505,), (134578743,), (134553208,), (134555349,),
-                              (134580057,), (134521306,), (134531547,), (134572084,), (134578576,), (134554727,),
-                              (134576873,), (134580336,), (134521331,), (134550133,), (134521336,), (134529017,),
-                              (134524668,), (134577049,), (134547258,), (134577158,), (134524356,), (134524581,),
+                              (134521306,), (134572084,), (134578576,), (134554727,),
+                              (134576873,), (134521331,), (134550133,), (134521336,), (134529017,),
+                              (134524668,), (134577049,), (134577158,),
                               (134576974,), (134576909,),
+
+                              (134531547,),  # void usage(int status) - call exit
+                              (134547258,),  # void __argmatch_die() - call usage
+                              (134524568,),  # int decode_switches(int argc, char** argv) - call usage
+                              (134524356,),  # int decode_switches(int argc, char** argv) - call usage
+                              (134524581,),  # int decode_switches(int argc, char** argv) - call usage
+                              (134580057,),  # _obstack_begin_worker(...) - call obstack_alloc_failed_handler
+                              (134580661,),  # void print_and_abort() - call exit
+                              (134580336,),  # void _obstack_newchunk(...) - call obstack_alloc_failed_handler
```

Comment:
these entries are currently failing. tbh, at the moment i have no idea why. i've added some comments around them for my sanity. i can revert these changes if you don't like the inconsistency.

Reply:
nothing actually changed, *awesome*
def appreciate additional context.  thanks.